frontend/src/v4/routes/jobs/jobs.component.tsx

```diff
@@ -164,6 +197,29 @@ export class Jobs extends PureComponent<IProps, IState> {
 					/>
 				</UserManagementTab>
 				{containerElement && this.renderNewJobForm(containerElement)}
+				{isV5() && (
+					<>
+						<NewJobBottomButton onClick={openNewJobForm}>
+							<AddCircleIcon />
+							<FormattedMessage id="jobs.addJobButton" defaultMessage="Add new Job" />
+						</NewJobBottomButton>
+						<Panel
+							open={Boolean(panelBottomAnchorEl)}
+							anchorEl={panelBottomAnchorEl}
+							onClose={closeNewJobForm}
+							anchorOrigin={{
+								vertical: 'top',
+								horizontal: 'center',
+							}}
+							transformOrigin={{
+								vertical: 'center',
+								horizontal: 'center',
+							}}
+						>
+							{this.renderNewJobFormPanel({ closePanel: () => {} })}
```

Comment:
What's with the empty arrow function here?

Reply:
Because the `renderNewJobFormPanel` wants a prop called `closePanel`, but the `open` state of the `NewJobFormPanel` component is dictated by `panelBottomAnchorEl`